force-app/main/default/classes/extentionController.cls

```diff
@@ -0,0 +1,71 @@
+public class ExtentionController {
```

Comment:
For future - please name controller with smth that makes more sense, for example
PrimaryContactController

Reply:
Changed to PrimaryContactController